src/clock-weather-card.ts

```diff
@@ -452,6 +458,21 @@ export class ClockWeatherCard extends LitElement {
     return this.getWeather().attributes.temperature ?? null
   }
 
+  private getCurrentHumidity (): number | null {
+    if (this.config.humidity_sensor) {
+      const humiditeeSensor = this.hass.states[this.config.humidity_sensor] as HumiditySensor | undefined;
```

Comment:
`humiditeeSensor` typo

Reply:
there is an existing interface HumiditySensor in types, and i was using your getCurrentTemperature as a template, which used 'temperatueSensor' for the same reason i think, existing TemperatureSensor, so i just went with that :)